frontend/src/components/TopNavigation.tsx

```diff
@@ -0,0 +1,86 @@
+import { FunctionComponent, ReactNode } from 'react'
+import { Link } from 'react-router-dom'
+import { useAuth0 } from '@auth0/auth0-react'
+import DistributeAidLogo from './branding/DistributeAidLogo'
+import DropdownMenu from './DropdownMenu'
+import TruckIcon from './icons/TruckIcon'
+import CubeIcon from './icons/PackageIcon'
+import CogIcon from './icons/CogIcon'
+import UserIcon from './icons/UserIcon'
+import MobileNavigation from './navigation/MobileNavigation'
+import DesktopNavigation from './navigation/DesktopNavigation'
+
+export interface NavLinkItem {
+  path: string
+  label: ReactNode
+  icon?: ReactNode
+}
+
+const NAV_LINKS: NavLinkItem[] = [
+  {
+    path: '/offers',
+    label: 'Offers',
+    icon: <CubeIcon className="w-5 h-5 mr-2" />,
+  },
+  {
+    path: '/shipments',
+    label: 'Shipments',
+    icon: <TruckIcon className="w-5 h-5 mr-2" />,
+  },
+  {
+    path: '/admin',
+    label: 'Admin',
+    icon: <CogIcon className="w-5 h-5 mr-2" />,
+  },
+]
+
+interface Props {
+  /**
+   * If true, the user's information and "log out" button will be hidden.
+   * Use this prop when you want to show the page header while things are
+   * still loading.
+   */
+  hideControls?: boolean
+}
+
+/**
+ * A full-width element that sits at the top of the page. It displays the DA
+ * branding and a dropdown-menu with some account information.
+ */
+const TopNavigation: FunctionComponent<Props> = ({ hideControls }) => {
+  const { user, logout } = useAuth0()
+
+  return (
+    <header className="py-2 bg-da-navy-100 h-nav sticky top-0">
+      <div className="max-w-5xl px-4 mx-auto h-full flex items-center justify-between">
+        <MobileNavigation navLinks={NAV_LINKS} />
+        <div className="flex items-center">
+          <Link to="/" className="text-white" aria-label="Go to the home page">
+            <DistributeAidLogo className="block h-8" />
+          </Link>
+        </div>
+        <DesktopNavigation navLinks={NAV_LINKS} />
```

Comment:
The desktop and mobile nav are different enough that they warrant 2 components. They take the same list of links, though, which means they'll always be in sync.

Reply:
nice